tests/test_cli_main.py

```diff
@@ -79,13 +83,39 @@ def test_cli_version(capsys, option):
 
 
 @pytest.mark.parametrize("option", good_scenarios)
-def test_main_run(capsys, option):
+def test_main_run(capsys, option, monkeypatch):
     """Test main cli command"""
-    with pytest.raises(SystemExit):
-        cli.main(option)
 
-    out, err = capsys.readouterr()
-    assert err == ""
+    with tempfile.TemporaryDirectory(
+        prefix="QDT_test_cli_run_",
+        ignore_cleanup_errors=True,
+    ) as tmpdirname:
+        # customize QDT working folder and profiles destination folder
+        tmp_dir = Path(tmpdirname).joinpath(f"scenario_{good_scenarios.index(option)}")
+        environ[
+            "QDT_LOCAL_WORK_DIR"
+        ] = f"{tmp_dir.parent.joinpath('qdt_working_folder').resolve()}"
```

Comment:
`Path(tmpdirname).joinpath('qdt_working_folder').resolve()` would be easier to read.

Reply:
If you say that, why not. It was a way to have only one variable to manage (the second being defined by the first). Changed.